hasura/migrations/1575910000000_init/up.sql

```diff
@@ -8,81 +8,57 @@ CREATE TYPE account_type AS ENUM (
   'DISCOURSE'
 );
 
-CREATE TYPE player_rank AS ENUM (
-  'PLAYER',
-  'BRONZE',
-  'SILVER',
-  'GOLDEN',
-  'PLATINIUM',
-  'DIAMOND'
+CREATE TYPE enneagram_type AS ENUM (
+  'REFORMER',
+  'HELPER',
+  'ACHIEVER',
+  'INDIVIDUALIST',
+  'INVESTIGATOR',
+  'LOYALIST',
+  'ENTHUSIAST',
+  'CHALLENGER',
+  'PEACEMAKER'
```

Comment:
Now that we know the enum types arent great to use, should we make this a table too with many to one relationship? That way we can have a description and other metadata for each type?

Reply:
Not sure that's a good idea, this will never change and therefore we don't really need to put these metadata it in our database

Reply:
no worries. Feel free to merge after resolving conflicts